src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -103,7 +103,93 @@ Sentry.init({
 });
 ```
 
-Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry. Here's an example for your server component:
+Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry.
+
+### Using a Tunneling Server
+
+Here's an example for your server component:
```

Comment:
```suggestion
Here's an example of how to set up an endpoint on your server that tunnels events to Sentry:
```

"Server component" is becoming a bit of an ambiguous term so we should avoid it in the wrong context.